docs/articles/oso/how-to-open-source-software.rst

```diff
@@ -0,0 +1,28 @@
+:orphan:
+
+How to open source software
+===========================
+
+.. tagslist:: howto
+
+When open sourcing source code there are some steps that needs to be completed.
+The purpose of these steps are to ensure that the code has a sound license, the
+source code repository is understandable and that there are clear descriptions
+on how to contribute. Following is a listing of these steps:
+
+#. Make sure that the following files are present in the root of the source code
+   repository:
+    * LICENSE: Containing the desired license for the code
```

Comment:
Ensuring a license is there is meaningless if it doesn't correspond with copyright. The first, and maybe most important step, is to ensure that there is copyright authority. Something similar to Debian's "copyright" file for example. We can of course do something different, but we need to know the code has copyright, the copyright is correctly assigned, and the copyright holder allows publication. The license is, of course, the key to the last part, but it needs to be grounded in copyright since it is meaningless on its own.

Reply:
Yes you are correct. Although this line states the point of the actual LICENSE file, which is to state the desired license. If there is copyright majority etc is up to the OSO while scanning etc. See stages below.

Reply:
Also we should add a step which makes sure there are copyright notices in the files...